docs/GettingStarted.md

```diff
@@ -1613,6 +1613,16 @@ require 'ddtrace'
 # Configure default RSpec integration
 Datadog.configure do |c|
   c.use :rspec, options
+  c.tracer writer_options: { buffer_size: 5000, flush_interval: 0.5 }
+end
+```
+
+If you have many fast unit tests, you will need to tweak flushing settings. You can enable `health_metrics` to tweak the numbers. This will send a metric called `datadog.tracer.queue.dropped.traces`
+
+```ruby
+Datadog.configure do |c|
+  c.use :rspec, options
+  c.diagnostics.health_metrics.enabled = true
```

Comment:
we should probably mention it also in the cucumber section.

Reply:
I removed all changes regarding cucumber for now. We will come back to this in the next weeks